src/protocol/abi/receipts.md

```diff
@@ -23,6 +23,8 @@ All receipts will have a `type` property:
   - [Panic](#panic-receipt)
   - [Revert](#revert-receipt)
   - [Log](#log-receipt)
+  - [Mint](#mint-receipt)
```

Comment:
We also need to update `ReceiptType` in the `src/protocol/tx_format/transaction.md`